project/CRUD/roles.php

```diff
@@ -145,6 +160,7 @@
         while ($row = mysqli_fetch_assoc($results)) {
             $id = $row['role_id'];
             $role_name = $row['role_name'];
+            $role_description = $row['role description'];
```

Comment:
Warning: Undefined array key "role description" in "..\CRUD\roles.php" on line 163

Please do check into the update section